src/components/views/right_panel/PinnedMessagesCard.tsx

```diff
@@ -155,12 +155,23 @@ const PinnedMessagesCard = ({ room, onClose }: IProps) => {
 
         // show them in reverse, with latest pinned at the top
         content = pinnedEvents.filter(Boolean).reverse().map(ev => (
-            <PinnedEventTile key={ev.getId()} room={room} event={ev} onUnpinClicked={onUnpinClicked} />
+            <PinnedEventTile key={ev.getId()} room={room} event={ev} onUnpinClicked={() => onUnpinClicked(ev)} />
         ));
     } else {
-        content = <div className="mx_RightPanel_empty mx_PinnedMessagesCard_empty">
-            <h2>{_t("You’re all caught up")}</h2>
-            <p>{_t("You have no visible notifications.")}</p>
+        content = <div className="mx_PinnedMessagesCard_empty">
+            <div>
+                <div className="mx_PinnedMessagesCard_MessageActionBar">
+                    <div className="mx_MessageActionBar_maskButton mx_MessageActionBar_reactButton" />
+                    <div className="mx_MessageActionBar_maskButton mx_MessageActionBar_replyButton" />
+                    <div className="mx_MessageActionBar_maskButton mx_MessageActionBar_optionsButton" />
```

Comment:
could do with an `XXX: We reuse the classes for simplicity, but deliberately not the components for non-interactivity.`